iphone/Classes/TiRootViewController.m

```diff
@@ -316,42 +316,56 @@ - (UIImage*)defaultImageForOrientation:(UIDeviceOrientation) orientation resulti
     *imageIdiom = UIUserInterfaceIdiomPhone;
     // Default
     image = nil;
-    if ([TiUtils isRetinaHDDisplay]) {
+  
+    // iPhone X
+    if ([TiUtils isRetinaiPhoneX]) {
+      if (UIDeviceOrientationIsPortrait(orientation)) {
+        // Portrait
+        image = [UIImage imageNamed:@"LaunchImage-1100-Portrait-2436h@3x"];
+      } else if (UIDeviceOrientationIsLandscape(orientation)) {
+        // Landscape
+        image = [UIImage imageNamed:@"LaunchImage-1100-2436h@3x"];
```

Comment:
Hi @hansemannn , any reason for the image not to have `landscape` in name? Thnx!

Reply:
Indeed. Xcode seems to store it as `LaunchImage-1100-2436@3x.png` inside the bundle, see this image from my [post at StackOverflow](https://stackoverflow.com/questions/46263795/iphone-x-incorrect-launch-screen-orientation-used):

<img src="https://i.stack.imgur.com/kOXsJ.png" />

Reply:
Which is incorrect, but it seems the way Xcode does it. I opened a radar already and awaiting feedback from Apple today.